app/controllers/api/v1/moves_controller.rb

```diff
@@ -90,7 +90,9 @@ def find_move
       end
 
       def import_moves_from_nomis
-        Moves::NomisSynchroniser.new(locations: from_locations, date: date).call
+        PaperTrail.request(whodunnit: nil) do
+          Moves::NomisSynchroniser.new(locations: from_locations, date: date).call
+        end
```

Comment:
:+1: to removing the current user from synchroniser changes. Very neat. Would it be worth adding an intentional comment saying why we decided not to associate those changes with a user?

Reply:
Yes agreed. Big fan of intention comments, not always perfect at writing them